tox.ini

```diff
@@ -65,3 +65,17 @@ ignore = E123,E125,E203,E402,E501,E741,F401,F811,F841,W503
 max-line-length = 160
 builtins = _
 exclude = .git,.tox,tests/unit/compat/
+
+[testenv:prepare_release]
+
+deps = 
+  antsibull-changelog
+  {[testenv:add_docs]deps}
+
+commands =
+
+  antsibull-changelog release --verbose --version '{env:RELEASE_VERSION}'
+  {[testenv:add_docs]commands}
```

Comment:
We do not need to use add_docs. This https://github.com/ansible-collections/amazon.cloud/pull/74/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R17-R19 can be deleted.